docs/platform/9_tips-and-tricks.md

```diff
@@ -29,6 +29,10 @@ For more information check out
 * [Grid Artifacts](https://docs.grid.ai/features/runs/artifacts)
 * [Lightning Loading Checkpoints](https://pytorch-lightning.readthedocs.io/en/latest/common/weights_loading.html)
 
+### Stop a Run after X hours
+As a convenience, we provide a [GitHub Action](https://github.com/sunitaprakash/gridai-stop-run) for stopping a Run that has been running for X no of hours.
```

Comment:
please set this to a Grid org repo. If you don't have permissions to do this I can for you.